tests/test_interactive_importer.py

```diff
@@ -1,8 +1,45 @@
 import unittest
 
+from bs4 import BeautifulSoup
+
 from gourmand.importers import interactive_importer
 
 
+class TestGetImages(unittest.TestCase):
+
+    def test_get_images(self):
+        self.html = """
```

Comment:
No need to assign `html` and `soup` to the class instance - just use `html = ...`  and `soup = ...`.